src/components/AnnotationEditor.jsx

```diff
@@ -8,11 +8,19 @@ import markdownRendererOptions from '../util/markdown-renderer-options';
 import markdownLogo from '../../res/markdown-logo.svg';
 
 const styles = {
+  'bottomContainer': {
+    display: 'flex',
+    'flexFlow': 'row wrap',
+  },
+  'button': {
+    flex: '1 1 auto',
```

Comment:
Would you please change these short hands to explicitly property declarations, each on their own line?

I'm not a fan of CSS shorthand notations. I nearly always have to take time to look up documentation on the shorthand if I need to modify it, because I cannot remember what it refers to (e.g. https://www.w3schools.com/cssref/css3_pr_flex.asp). I strongly prefer actually specifying each property explicitly on its own line.